docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -159,6 +159,20 @@ See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getA
 <2> Use the assertion library provided by the Synthetics agent to look for the
 expected value. See https://jestjs.io/docs/expect[Jest expect docs] for more information.
 
+[discrete]
+[[synthetics-create-test-script-recorder]]
+=== Synthetics Recorder
+
+beta[]
+
+Use the Synthetics Recorder to generate code for one or more steps by interacting with a web page directly.
+
+// In what circumstances should you use Synthetics Recorder?
```

Comment:
> // In what circumstances should you use Synthetics Recorder?

The recorder is intended for use any time a user would like to create a repeatable synthetic browser interaction without having to actually write code themselves.